.github/workflows/release-new-version.yml

```
@@ -66,6 +66,17 @@ jobs:
         run: |
           git config --local user.name 'github-actions'
           git config --local user.email '41898282+github-actions[bot]@users.noreply.github.com'
+          # maybe we need to move this package.json as we need it in multiple workflows
+          npm install -g 'git+ssh://git@github.com:knutties/newman.git#37708c1fbf6b36240ac796a42dc5a7b435990764'
+          POSTMAN_DIR=postman/collection-dir
+          POSTMAN_JSON_DIR=postman/collection-json
+          # generate postman collections for all postman directories
+          for connector_dir in ${POSTMAN_DIR}/*
+          do
+            connector=$(basename ${connector_dir})
+            newman dir-import ${POSTMAN_DIR}/${connector} -o ${POSTMAN_JSON_DIR}/${connector}.postman_collection.json
+          done
+          (git diff --quiet && git diff --staged --quiet) || (git commit -am 'test(postman): updated postman collection files' && echo "Committed changes") || (echo "Unable to commit following changes:" && git diff)
```

Nit (commit message): `test(postman): update postman collection files`

```suggestion
          (git diff --quiet && git diff --staged --quiet) || (git commit -am 'test(postman): update postman collection files' && echo "Committed changes") || (echo "Unable to commit following changes:" && git diff)
```

Also, any specific reason why you're combining multiple commands with `||` instead of, say if-else?

---

Shell's boolean is a little different than normal boolean - felt a single line makes it readable than a branch representation i.e check and move to the next command.

---

```suggestion
          (git diff --quiet && git diff --staged --quiet) || (git commit -am 'test(postman): update postman collection files' && echo "Committed changes") || (echo "Unable to commit the following changes:" && git diff)
```
This would make it more readable? Added `the` to be specific about the changes done.